src/components/lower-info-area/name-build-sub-components/unique-names.vue

```diff
@@ -31,6 +31,9 @@
         <strong>Westcoast Barbers Inc.</strong>
       </p>
     </v-row>
+    <v-row class="ml-1 mb-4">
+      <a href="./pdfs/groupings_of_similar_descriptive_words.pdf"> Groupings of similar descriptive words</a>
```

Comment:
Why do you have a space in front of the text?

Reply:
It was a mistake. corrected.

Reply:
OK. If it was intentional then I had a comment about it.

Resolved.